wallets/provider-clover/src/namespaces/solana.ts

```diff
@@ -0,0 +1,49 @@
+import type { CaipAccount } from '@rango-dev/wallets-core/namespaces/common';
+import type { SolanaActions } from '@rango-dev/wallets-core/namespaces/solana';
+
+import { NamespaceBuilder } from '@rango-dev/wallets-core';
+import { LegacyNetworks } from '@rango-dev/wallets-core/legacy';
+import { builders as commonBuilders } from '@rango-dev/wallets-core/namespaces/common';
+import {
+  builders,
+  CAIP_NAMESPACE,
+  CAIP_SOLANA_CHAIN_ID,
+} from '@rango-dev/wallets-core/namespaces/solana';
+import { CAIP } from '@rango-dev/wallets-core/utils';
+
+import { WALLET_ID } from '../constants.js';
+import { solanaClover } from '../utils.js';
+
+const connect = builders
```

Comment:
Solana namespace also need to have `changeAccount` functionality. to reproduce you can only connect solana namespace and then try to switch account in the wallet.

Reply:
I added a custom `changeAccountSubscriber` to the Clover actions.